app/components/Input.tsx

```diff
@@ -120,6 +121,8 @@ export type Props = React.InputHTMLAttributes<
   flex?: boolean;
   short?: boolean;
   margin?: string | number;
+  error?: string;
+  preview?: string;
```

Comment:
I think it's overkill to add this in here, can just add below the `Input` element